pulp_npm/app/serializers.py

```diff
@@ -79,7 +68,7 @@ class Meta:
         help_text="The policy to use when downloading content. The possible values include: "
         "'immediate', 'on_demand', and 'streamed'. 'immediate' is the default.",
         choices=core_models.Remote.POLICY_CHOICES,
-        default=core_models.Remote.IMMEDIATE,
```

Comment:
why is the default not needed anymore?

Reply:
https://discourse.pulpproject.org/t/be-careful-with-default-values-in-drf/1910

Reply:
thank you! that makes sense